packages/replayio/src/utils/prompt/prompt.ts

```diff
@@ -0,0 +1,38 @@
+import { readFromCache, writeToCache } from "../cache";
+import { promptHistoryPath } from "./config";
+import { PromptHistory } from "./types";
+
+export async function prompt(id?: string): Promise<boolean> {
+  return new Promise(resolve => {
+    const stdin = process.stdin;
+    const wasRaw = stdin.isRaw;
```

Comment:
nit: `wasRaw` suggests that it we know it has been `true` - maybe `oldRaw`/`prevRaw` would fit this better?
```suggestion
    const oldRaw = stdin.isRaw;
```